licenses/WidePix

```diff
@@ -0,0 +1,13 @@
+WidePix
+
+© 2020 Nash Muhandes
+
+LICENSE:
+
+You MAY make derivative works based on mine - HOWEVER:
+
+- You may not sell this work
+
+- Your work must comply with id Software's original licenses (i.e. they must only work on the game from which they originated)
+
+- All derivative works must credit the original authors (id Software (Doom), Raven Software (Hexen/Heretic), Rogue Software (Strife), Digital Café (Chex), Nash Muhandes)
```

Comment:
I don't see anything in that license that would permit redistribution, so mirror and bindist restrictions are asked for.

Reply:
It might be worth asking upstream to clarify this point as that probably wasn't their intention. I'm not sure whether that's compatible with the original licenses though.

Reply:
I'm contacting upstream to see if they can update the license to be more explicit.

Reply:
Upstream has updated the WidePix license to permit distribution: https://github.com/nashmuhandes/WidePix/commit/92738042ca3a37f28153a09809d80a7d61090532

Reply:
Still not a free license because of the "no sell" clause, but at least we won't need mirror and bindist restrictions.

Reply:
Yes, unfortunately GZDoom widescreen graphics will have to remain part of the non-free USE flag. I suppose that's to be expected given the similar "no sell" clause of the parent Doom asset license. :disappointed: